dispatcher/messages.py

```
@@ -0,0 +1,225 @@
+
+    if res.status_code not in [200, 204]:
+        raise Exception(
+            f"Failed to update {resource_endpoint} {obj_id}: {res.status_code}, {res.text}"
+        )
+
+    # Return JSON response if available, else construct a dict with the provided values.
+    return res.json() if res.text else {"id": obj_id, **values}
+
+
+def update_objects(
+    message: dict, resource_endpoint: str | None = None, selector_mapping: dict = {}
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)